src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAwaitForeachTests.cs

```diff
@@ -2384,6 +3411,72 @@ public S(int i)
 
             comp = CreateCompilationWithTasksExtensions(source, options: TestOptions.ReleaseExe);
             CompileAndVerify(comp, expectedOutput: expectedOutput, verify: Verification.FailsILVerify).VerifyDiagnostics();
+
+            var runtimeAsyncComp = CodeGenAsyncTests.CreateRuntimeAsyncCompilation(source.Replace(" + s_IAsyncEnumerable", ""));
```

Comment:
It looks like it was likely meant to do something like `Replace(s_IAsyncEnumerable, "")` here.